XAML Essentials/XAML Essentials/QuotesPage.xaml.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+using Xamarin.Forms;
+using Xamarin.Forms.Xaml;
+
+namespace XAML_Essentials
+{
+    [XamlCompilation(XamlCompilationOptions.Compile)]
+    public partial class QuotesPage : ContentPage
+    {
+        string[] quotes = new string[] { "Hôm nay là một ngày đẹp trời!",
+                                         "Baby Shark Duu Duu Duu...",
+                                         "Build a simple app for browsing a list of quotes" };
+        private int index = 0;
+        public QuotesPage()
```

Comment:
string[] quotes = new string[] { "Hôm nay là một ngày đẹp trời!",
                                         "Baby Shark Duu Duu Duu...",
                                         "Build a simple app for browsing a list of quotes" };
        private int index = 0;
        public QuotesPage()

3 cách khai báo này có gì khác nhau không e

Reply:
khác nhau về khả năng truy cập ạ
với khai báo 1 là khai báo mặc định Default thì Mảng string  thì nó được định nghĩa là biến toàn cục chỉ có thể truy cập trong lớp.
với khai báo 2 là khai báo private thì biến index sẽ chỉ được truy cập trong lớp chứa nó.
với khai báo 3 là khai báo public thì phương thức, biến có thể truy cập ở mọi nơi.